.tekton/integration-tests/integration-test-scenarios/compliance-operator-fbc-4-18-integration-tests-arm-mixarch.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  annotations:
+    test.appstudio.openshift.io/optional: "true"
+  name: compliance-operator-fbc-v4-18-e2e-tests-amd
```

Comment:
Should be named as `-arm-mixarch, but it is not consistently named.

Reply:
arm-mixarch means the control plane is based on ARM and compute nodes are based on AMD64. So I would prefer to keep the "mixarch" and update the name to make it consistent.